drivers/espi/espi_it8xxx2.c

```diff
@@ -696,11 +696,18 @@ static int espi_it8xxx2_send_vwire(const struct device *dev,
 		(struct espi_vw_regs *)config->base_espi_vw;
 	uint8_t vw_index = vw_channel_list[signal].vw_index;
 	uint8_t level_mask = vw_channel_list[signal].level_mask;
+#ifdef CONFIG_ESPI_SET_VW_VALID_WITH_LEVEL
+	uint8_t valid_mask = vw_channel_list[signal].valid_mask;
+#endif
 
 	if (signal > ARRAY_SIZE(vw_channel_list)) {
 		return -EIO;
 	}
 
+#ifdef CONFIG_ESPI_SET_VW_VALID_WITH_LEVEL
+	vw_reg->VW_INDEX[vw_index] |= valid_mask;
```

Comment:
Use `espi_vw_set_valid()` instead.  Also consider whether this can be set once at initialization time.

Reply:
Dear sir,

We've modified it again to remove the espi_vw_set_valid() and put the valid bits settings together with the espi_it8xxx2_send_vwire(). Meanwhile, the ESPI_SET_VW_VALID_WITH_LEVEL has been removed as well.

Thanks for guidance & regards,
BJ